metapool/sample_sheet.py

```diff
@@ -21,26 +28,31 @@
 SHEET_TYPES = (_STANDARD_METAG_SHEET_TYPE, _ABSQUANT_SHEET_TYPE,
                _STANDARD_METAT_SHEET_TYPE)
 
+_BASE_BIOINFORMATICS_COLS = [
+    'Sample_Project', 'QiitaID', 'BarcodesAreRC', 'ForwardAdapter',
+    'ReverseAdapter', 'HumanFiltering', 'library_construction_protocol',
+    'experiment_design_description']
+_BIOINFORMATICS_COLS_W_REP_SUPPORT = \
+    _BASE_BIOINFORMATICS_COLS + ['contains_replicates']
+_CONTACT_COLS = ['Sample_Project', 'Email']
+_SAMPLE_CONTEXT_COLS = [SAMPLE_NAME_KEY, SAMPLE_TYPE_KEY,
+                        PRIMARY_STUDY_KEY, SECONDARY_STUDIES_KEY]
+
```

Comment:
I am not sure I want to make this change until I am going to go through and replace *ALL* the instances of, e.g., 'Assay' within this module with `ASSAY`. @charles-cowart  how do you feel about making this change now?  So far my goal has been to make minimal changes (since I'm playing in someone else's sandbox, and nervous about it) but I am willing to make these changes throughout if you are on board with them.

Reply:
Sure! Please feel free!

Reply:
Done